roles/containers/tasks/main.yml

```diff
@@ -47,6 +47,35 @@
       AMQP_URL: "{{ amqp_protocol }}://{{ rabbitmq_username }}:{{ rabbitmq_password }}@{{ rabbitmq_host }}/{{ rabbitmq_vhost | urlencode | replace('/', '%2F') }}"
       DATABASE_URL: "ecto://{{ wanda_postgres_user }}:{{ wanda_postgres_password }}@{{ wanda_postgres_host }}/{{ wanda_postgres_db }}"
 
+- name: Prepare env variables for Web container
+  ansible.builtin.set_fact:
```

Comment:
I'm not sure at all if setting this values as facts is a good idea.
When you set values as fact, they can be retrieved later on from the main ansible host machine, etc.
I mean, they have their usage. Here, you are using them as simple value holders.
Personally, I would put them all directly in the container env entry.

I know that you did put some effort on this, but I don't think we should use it.

Reply:
Sure, I just wanted to skip passing empty env's to the docker container. 
I reverted the changes :+1: